packages/lisk-transactions/src/index.ts

```diff
@@ -47,12 +36,8 @@ export {
 	VoteTransaction,
 	InTransferTransaction,
 	transfer,
-	registerDelegate,
```

Comment:
we are no longer exporting create functions other than `transfer` and `registerMultisignature`?

Reply:
Done. Resolving.